docs/rules/custom/forbidden-function-call.md

```diff
@@ -0,0 +1,49 @@
+# forbidden-function-call
+
+**Summary**: Forbidden function call
+
+**Category**: Custom
+
+## Description
+
+This custom rule allows providing Regal a list of
+[built-in functions](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions) that should be
+considered forbidden. Any call to a function in the list will be considered a violation.
+
+Another, more advanced, option to achieve the same result is the
+[capabilities](https://www.openpolicyagent.org/docs/latest/deployments/#capabilities) feature in OPA. While a more
+capable option, it is also more demanding to configure and maintain. If you're already using the capabilities feature
```

Comment:
Worth explaining that it's more capable as it's enforced at build time? or are you referring to something else.

I guess it also works for different OPA versions? Not sure what else might be being referred to here.

Reply:
It is more capable as it allows you to:

- Add new functions that OPA should be aware of, Regal only forbids.
- Disable certain things like some features not necessarily being built-in functions, like "future" keywords.
- List allowed hosts in network calls.

I can amend the docs to explain this.

Reply:
Updated with list above, and changed the "build pipeline" sentence as well.